src/endpoints/auth/check-auth.route.ts

```diff
@@ -0,0 +1,48 @@
+import { jwtConfig } from '../../shared/configs';
+import { USER_STATE_COOKIE } from '../../shared/consts';
+import { UserEntity } from '../../shared/database';
+import { UserStateEnum } from '../../shared/enums';
+import { UnauthorizedException } from '../../shared/errors';
+import { JWTService } from '../../shared/services';
+import { RouteCustomOptions } from '../../shared/types';
+import { checkAuthSchema, ICheckAuthBodySchema as ICheckAuthBodyInput } from './schemas';
+
+const { secret, accessDeathDate, refreshDeathDate } = jwtConfig;
+
+export const checkAuthRoute: RouteCustomOptions<{ Body: ICheckAuthBodyInput }> = {
+  url: '/check-auth',
```

Comment:
I don't like the naming. It's not about checking auth, it's about refreshing tokens. Try something like `auth/refresh-tokens` or `auth/refresh`